Mathlib/Tactic/DefEqTransformations.lean

```diff
@@ -114,6 +114,57 @@ elab "reduce" loc?:(ppSpace Parser.Tactic.location)? : tactic =>
   runDefEqTactic (fun _ e => reduce e (skipTypes := false) (skipProofs := false)) loc? "reduce"
 
 
+/-! ### `unfold_let` -/
+
+/-- Unfold all the fvars from `fvars` in `e` that have local definitions (are "let-bound"). -/
+def unfoldFVars (fvars : Array FVarId) (e : Expr) : MetaM Expr := do
+  transform (usedLetOnly := true) e fun node => do
+    match node with
+    | .fvar fvarId =>
+      if fvars.contains fvarId then
+        if let some val ← fvarId.getValue? then
+          return .visit (← instantiateMVars val)
+        else
+          return .continue
+      else
+        return .continue
+    | _ => return .continue
+
+/--
+This tactic is subsumed by the `unfold` tactic.
+
+`unfold_let x y z at loc` unfolds the local definitions `x`, `y`, and `z` at the given
+location, which is known as "zeta reduction."
+This also exists as a `conv`-mode tactic.
+
+If no local definitions are given, then all local definitions are unfolded.
+This variant also exists as the `conv`-mode tactic `zeta`.
+-/
+@[deprecated unfold]
+syntax (name := unfoldLetStx) "unfold_let" (ppSpace colGt term:max)*
+  (ppSpace Parser.Tactic.location)? : tactic
+
+elab_rules : tactic
+  | `(tactic| unfold_let $[$loc?]?) => do
+    logWarning "The `unfold_let` tactic is deprecated. Please use `unfold` instead."
```

Comment:
Should this mention `zeta` instead? Same below, for conv mode.

Reply:
Not sure what you would like to see...